docs/BUILD.md

```diff
@@ -201,7 +201,7 @@ cmake -g 'Visual Studio 16 2019' -A x64 ..
 
 ```
 mkdir build && cd build
-cmake -g 'Linux Makefiles' ..
+cmake -G 'Unix Makefiles' ..
```

Comment:
This might break things...

Reply:
No, our docs were wrong. This was the same build fix we did months ago, we must have just forgot to update the docs.